doc/control-plane-events.md

```diff
@@ -1,101 +1,7 @@
 # Control Plane Events
 
 ## Introduction
-Control plane events have been added to enable the device service to emit events onto the message bus when a device has been added, updated, or deleted. This is in lieu of official Control Plane Events (CPE) in an upcoming release of EdgeX. Once EdgeX officially supports CPE, these events will be dropped in favor of those.
+Control plane events have been added to enable the device service to emit events onto the message bus when a device has been added, updated, or deleted.
 
-See edgex-docs [issue #581](https://github.com/edgexfoundry/edgex-docs/issues/581)
+See: https://docs.edgexfoundry.org/2.3/microservices/core/metadata/Ch-Metadata/#device-system-events
```

Comment:
FYI, the docs now have the ability to alias the latest docs version with `latest` which currently is set to `2.2` . Doesn't help here since you need 2.3, but once we release in the fall we want to update the link to use `latest` instead of `2.3`.